dot/parachain/availability-store/messages.go

```diff
@@ -6,15 +6,14 @@ package availabilitystore
 import (
 	"errors"
 	"fmt"
-	"time"
 
 	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/trie"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
-var errInvalidErasureRoot = errors.New("Invalid erasure root")
+var ErrInvalidErasureRoot = errors.New("Invalid erasure root")
```

Comment:
does this need to be public? I don't see any references to it outside of this package.

Reply:
Updated.